src/angularjs/src/app/places/list/place-list.html

```diff
@@ -10,20 +10,32 @@ <h2 class="sidebar-title">City/Town Search</h2>
         <div class="row">
             <div class="column">
                 <div class="form-group">
-                    <label for="1">Filter by city or town name</label>
-                    <input type="text" class="form-control">
+                    <label for="neighborhood-filter">Filter by city or town name</label>
+                    <ui-select
+                        name="neighborhood-filter"
+                        ng-model="placeList.neighborhoodFilter"
+                        title="Filter by Neighborhood">
+
+                        <ui-select-match allow-clear="true" placeholder="Neighborhood" >
+                            {{$select.selected.label}}
+                        </ui-select-match>
+                        <ui-select-choices repeat="neighborhood.uuid as neighborhood in
```

Comment:
Hm, okay. This is basically the same dropdown as used by the admin portion of the site. For now at least, the list is limited to neighborhoods owned/uploaded by the user's organization, although for this public page, the set of neighborhoods available should be different.